index.js

```diff
@@ -0,0 +1,279 @@
+module.exports.fhirconvert = function(str) {
+    /**
+     * Full fhirconvert function: validates, converts, then identifies variables.
+     * @returns converted fhirpath expression
+     */
+    if (validate(str)){
+        return varfind(convert(str));
+    }
+    else {
+        return null;
+    }
+
+    /**
+     * Verifies normal syntax by confirming var names, function names,
+     * and number of parenthesis.
+     * @param {string} str - inputted normal syntax expression
+     * @returns boolean, valid or invalid
+     */
+    function validate(str) {
+        var vars = ["a", "b" ,"c", "d", "e", "f", "g", "h", "i"];
```

Comment:
I think it was already mentioned but this should be provided by the library users.